apps/betterangels-backend/clients/tests/test_permissions.py

```diff
@@ -266,14 +266,17 @@ def test_view_client_documents_permission(self, user_label: str, should_succeed:
         query = """
             query ViewClientDocuments {
                 clientDocuments {
-                    id
+                    totalCount
```

Comment:
we don't need to include `totalCount` here. It's not hurting anything, but it's not being used either 🤷